src/fhir/generator/json/definitions.json/dataelements.json

```diff
@@ -80340,7 +80340,7 @@
                         "max": "1",
                         "type": [
                             {
-                                "code": "code"
+                                "code": "CodeableConcept"
```

Comment:
Can we please add a fix for Condition resource too. Or you want to split to another ticket?

Reply:
No problem adding Condition fix also.